workbench/backend/http_requests/fetch_plugins.http

```diff
@@ -0,0 +1,12 @@
+GET https://qt.fsm-dev.cloud.sap/optimization/api/v1/plugins
```

Comment:
This is a http request template for IntelliJ. Please add some documentation into ReadMe on how to use it and how to provide the values for it (use a configuration)